crypto/hashtable/hashtable.c

```diff
@@ -171,6 +171,7 @@ static struct ht_neighborhood_st *alloc_new_neighborhood_list(size_t len,
             return NULL;
     }
     memset(ret, 0, sizeof(struct ht_neighborhood_st) * len);
+    fprintf(stderr, "Allocating new neighborhood %p(%p)\n", (void *)ret, (void *)*freeptr);
```

Comment:
printf debugging is grand but oughtn't be merged :grin:

Reply:
its a draft PR, no intention to merge this, its just here to iterative help debug an issue for @rsbecker